content/en/docs/Tasks/Troubleshooting/clusterserviceversion.md

```diff
@@ -11,7 +11,7 @@ description: >
 
 If the OLM operator encounters an unrecoverable error when attempting to install the operator, the `CSV` will be placed in the `failed` phase. The OLM operator will constantly update the `Status` with useful information regarding the state of the `CSV`. You can check the `Status` of your `CSV` with the following command:
 
-`$ kubectl -n my-catalogsource-namespace get csv prometheusoperator.0.32.0 -o yaml | yq r - status`
+`$ kubectl get csv prometheusoperator.0.32.0 -n <namespace> -o yaml | yq e '.status' -`
```

Comment:
I think this command depends entirely on what yq client you're using - do we specify which yq client is used throughout the documentation?